docs/datatypes.rst

```diff
@@ -0,0 +1,65 @@
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`cal::local_datetime`    | ``java.time.LocalDateTime``                         |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`cal::relative_duration` | ``com.edgedb.driver.datatypes.RelativeDuration`` \* |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`datetime`               | ``java.time.OffsetDateTime`` \*                     |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`duration`               | ``java.time.Duration``                              |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`float32`                | ``float``                                           |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`float64`                | ``double``                                          |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`int16`                  | ``short``                                           |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`int32`                  | ``int``                                             |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`int64`                  | ``long``                                            |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`bigint`                 | ``java.math.BigInteger``                            |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`decimal`                | ``java.math.BigDecimal``                            |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`json`                   | ``com.edgedb.driver.datatypes.Json``                |
++------------------------------------+-----------------------------------------------------+
+| :eql:type:`uuid`                   | ``UUID``                                            |
++------------------------------------+-----------------------------------------------------+
+| :eql:func:`range`                  | ``com.edgedb.driver.datatypes.Range``               |
++------------------------------------+-----------------------------------------------------+
+
+\* For ``cal::relative_duration``, ``Period`` and ``Duration`` are implicitly 
+converted to and from ``RelativeDuration``.
+
+\* For ``std::datetime``, both ``java.time.OffsetDateTime`` and 
+``java.time.ZonedDateTime`` are compatable.
```

Comment:
Will these be rendered as different symbols automatically? If not, maybe use [rst footnotes](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#footnotes) instead.